src/components/toolbar/tile-toolbar.tsx

```diff
@@ -84,19 +61,18 @@ export const TileToolbar = observer(
     // when you click in the tile, that would be super responsive.
     if (!enabled) return(null);
 
-    const buttons = buttonDescriptions.map((desc) => {
+    const buttons = buttonDescriptions.map((desc, i) => {
       if (isValidButtonDescription(desc)) {
         const buttonHasArg = !(typeof desc === 'string');
         const name = buttonHasArg ? desc[0] : desc;
         const info = getToolbarButtonInfo(tileType, name);
         if (info) {
           const Button = info?.component;
-          const buttonElt = buttonHasArg ? <Button name={name} args={desc}/> : <Button name={name}/>;
-          const tooltip = formatTooltip(desc, info);
-          return (
-            <Tooltip key={name} title={tooltip} {...tipOptions} >
-              {buttonElt}
-            </Tooltip>);
+          if (buttonHasArg) {
+            return (<Button key={i} name={name} args={desc}/>);
+          } else {
+            return (<Button key={i} name={name}/>);
```

Comment:
I'd think it'd be better to use `key={name}` here. It is typically not good to use an index for the key incase the order changes. But in this case I don't think the order would ever change without the whole component being re-rendered anyway.  It might also be that you could drop the key since at least as far as I've seen, each of the buttons is a different component type anyway.

Reply:
I switched to an index since I we can now have buttons configured like `[name arg]`, so I was worried that the name might not be unique in the list.

Reply:
Then use something like ``key={`${i}-${name}`}`` so you're protected from ambiguous names or ordering changes (but not malicious ordering changes of elements with the same name 😉).